xml/System.Net.Sockets/SocketTaskExtensions.xml

```diff
@@ -414,12 +414,12 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="3" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="socket">To be added.</param>
-        <param name="buffer">To be added.</param>
-        <param name="socketFlags">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="socket">The socket to perform the receive operation on.</param>
+        <param name="buffer">A region of memory that is the storage location for the received data.</param>
+        <param name="socketFlags">A bitwise combination of the <see cref="T:System.Net.Sockets.SocketFlags" /> values.</param>
+        <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        <summary>Receives data from a connected socket.</summary>
+        <returns>A task that represents the asynchronous receive operation. The value of the <paramref name="TResult" /> parameter contains the number of bytes received.</returns>
```

Comment:
Looks like a typo referencing the `TResult` in `ValueTask<TResult>`, unless this is a docs feature.

```suggestion
        <returns>A task that completes with the number of bytes received, or 0 if the end of the stream has been reached.</returns>
```